api/scpca_portal/config/common.py

```diff
@@ -0,0 +1,156 @@
+import os
+from distutils.util import strtobool
+from os.path import join
+
+import dj_database_url
+from configurations import Configuration
+
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+class Common(Configuration):
+
+    INSTALLED_APPS = (
+        "django.contrib.admin",
+        "django.contrib.auth",
+        "django.contrib.contenttypes",
+        "django.contrib.sessions",
+        "django.contrib.messages",
+        "django.contrib.staticfiles",
+        # Third party apps
+        "django_extensions",
+        "rest_framework",  # utilities for rest apis
+        "django_filters",  # for filtering rest endpoints
+        "drf_yasg",
+        "computedfields",  # Allows for computed fields on models
+        "safedelete",  # soft-deletes objects
+        "corsheaders",
+        # Your apps
+        "scpca_portal",
+    )
+
+    # https://docs.djangoproject.com/en/2.0/topics/http/middleware/
+    MIDDLEWARE = (
+        "corsheaders.middleware.CorsMiddleware",
+        "django.middleware.security.SecurityMiddleware",
+        "django.contrib.sessions.middleware.SessionMiddleware",
+        "django.middleware.common.CommonMiddleware",
+        "django.middleware.csrf.CsrfViewMiddleware",
+        "django.contrib.auth.middleware.AuthenticationMiddleware",
```

Comment:
Mmm probably not but it's one of the things that django inits every project with. Not entirely sure if removing it will have any unintended consequences, but I could test it. You think it's worth it?